### PR TITLE
Fixed a few bugs

### PR DESCRIPTION
**Pull request by KanishkaHalder1771**
Eliminates any links redirecting to "https://medium.com/".
Eliminates the entry of the 'Featured Writer' and its link into the list.
Changed  encoding of 'output.txt' to 'utf8'

### CODE REVIEW

File: Grab_that_article_on_Medium.py [hunk 1]
```diff
@@ -104,19 +104,20 @@
 links = [i.get_attribute('href') for i in driver.find_elements_by_xpath("//a[@data-action-source]")]
 unique_list = list(OrderedDict(zip(links, repeat(None)))) # To remove duplicates from list
 
-f= open("output.txt","a+") # Stores ouput in output.txt in the same file directory
+f= open("output.txt","a+",encoding='utf8') # Stores ouput in output.txt in the same file directory
```
Comment (ZER-0-NE) [inline Grab_that_article_on_Medium.py]: Was the encoding necessary?

File: Grab_that_article_on_Medium.py [hunk 2]
```diff
@@ -104,19 +104,20 @@
...
         f.write(time.strftime("%Y-%m-%d %H:%M") + '\n')
         f.write(tag[i].text)
         f.write('\nLink is -->   ' + str(l)  + '\n\n')
 else:
-    for i in range(tag_len):
-        f.write(time.strftime("%Y-%m-%d %H:%M") + '\n')
-        f.write(tag[i].text)
-        l = tag[i].find_element_by_css_selector('a').get_attribute('href');
-        f.write('\nLink is -->   ' + str(l)  + '\n\n')
+    for i in range(52):
```
Comment (ZER-0-NE) [inline Grab_that_article_on_Medium.py]: Why 52?

**Comment by ZER-0-NE** [comment]
Thanks for the PR @KanishkaHalder1771 
Kindly make the PR title more informative from next time. 
"Fixed a few bugs" is very vague and it would be difficult to track in the future.

File: Grab_that_article_on_Medium.py (hunk 1 above)
Comment (KanishkaHalder1771) [reply Grab_that_article_on_Medium.py]: The script is not running without the encoding on my machine. I don't know whats the default encoding. But I don't think adding the 'utf8' encoding will create any problems.

File: Grab_that_article_on_Medium.py (hunk 2 above)
Comment (KanishkaHalder1771) [reply Grab_that_article_on_Medium.py]: Line 116 - `if i!=0 and i!=4:                                #Eliminating the blank 1st entry and 'Featured Writer'`
This line eliminates 2 entries from the list. Thus to maintain 50 entries in the list I wrote 52.
For i=0 the `tag[0].text` is null.
And for i=4 it gives a 'Featured Writer' link which is not actually an article.

File: Grab_that_article_on_Medium.py (hunk 1 above)
Comment (ZER-0-NE) [reply Grab_that_article_on_Medium.py]: Are you using something other than Linux or windows?
I have not checked it's working yet, so I want to know if it fails in the future.

Comment (pratikmishra356) [reply Grab_that_article_on_Medium.py]: The script is running without encoding and with encoding also  on Mac.
```suggestion
f= open("output.txt","a+",encoding='utf8') # Stores ouput in output.txt in the same file ### directory
```

**Pull request closed without merging**